bin/submission_new.py

```diff
@@ -218,14 +218,12 @@ def extract_top_metadata(self):
 		available_columns = [col for col in columns if col in self.metadata_df.columns]
 		return self.metadata_df[available_columns].to_dict(orient='records')[0] if available_columns else {}
 	def extract_biosample_metadata(self):
-		columns = ['bs_package','isolate','isolation_source','host_disease','host','collected_by','lat_lon',
-				   'sex','age','geo_location','organism','purpose_of_sampling', 'race','ethnicity','sample_type']  # BioSample specific columns
+		columns = ['bs_package','strain','isolate','host_disease','host','collected_by','lat_lon','geo_location','organism','sample_type','collection_date']  # BioSample specific columns
```

Comment:
I'm pretty sure we need isolation_source for some GenBank code I just added.
But if age, sex, race, ethnicity aren't breaking anything, let's leave them in too.  That way, if someone puts data there it will be ingested and uploaded.

Reply:
Added isolation_source, age, sex, race and ethnicity.